tests/test_aibs_data_set.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from ipfx.aibs_data_set import AibsDataSet
+
+
+@pytest.mark.parametrize('NWB_file', ['H18.03.315.11.11.01.05.nwb'], indirect=True)
+def test_validate_required_sweep_info(NWB_file):
+
+    sweep_info = [{"sweep_number": 0}]
+    dataset = AibsDataSet(sweep_info, nwb_file=NWB_file, api_sweeps=False)
+
+    print list(dataset.sweep_table) == dataset.FILTER_COLUMN_NAMES
```

Comment:
did you mean `assert` instead of `print`?

Reply:
good catch